README.md

```diff
@@ -1,239 +1,49 @@
+| `BITBUCKET_APP_USERNAME`    | N/A (Required)                  |                                                                                                                                                                                            |
+| `BITBUCKET_APP_PASSWORD`    | N/A (Required)                  |                                                                                                                                                                                            |
+| `BITBUCKET_API_URL`         | `https://api.bitbucket.org/2.0` |                                                                                                                                                                                            |
+| `BITBUCKET_HOSTNAME`        | `bitbucket.org`                 |                                                                                                                                                                                            |
+| `BITBUCKET_REPO_FULL_NAME`  | N/A (Required)                  | Path to repository. Usually in the format `<namespace>/<project>`. When running in bitbucket pipelines this is automatically provided.                                                                                                                         |
+| `BITBUCKET_BRANCH         ` | N/A (Optional)                  | Branch to fetch manifest from and open pull requests against. When running in bitbucket pipelines this is automatically provided.                                                                                                                            |
+| `GITHUB_ACCESS_TOKEN`       | N/A                             | An Github personal access token with the `pulic_repo` to increase API rate limits.                                                                                                         |
+
+## Setup
+
+1. Add the below to a `bitbucket-pipelines.yml` in the root of your repository
+```yml
+pipelines:
+  custom:
+    dependabot:
+      - step:
+          name: "Dependabot Scan"
+          script:
+            - pipe: docker://aligent/dependabot-pipe:latest
+              variables:
+                PACKAGE_MANAGER: "npm_and_yarn" # Replace as needed
+                BITBUCKET_APP_USERNAME: "$BITBUCKET_APP_USERNAME"
+                BITBUCKET_APP_PASSWORD: "$BITBUCKET_APP_PASSWORD"
+                GITHUB_ACCESS_TOKEN: "$GITHUB_ACCESS_TOKEN"
+```
+
+1. Create a Bitbucket App password [here](https://bitbucket.org/account/settings/app-passwords/new) with the below permissions
+
+![Dependabot Permissions](/doc/dependa_bot_permissions.png)
+
+1. Create a Github Access token [here](https://github.com/settings/tokens/new) with just the `public_repo` permission
```

Comment:
Can we create access tokens for an organisation instead of a user? If not, we might need to create a generic Aligent GitHub user (if we don't have one already).

Reply:
Doesn't look like it unfortunately :( I don't think we have an aligent github user, but we did have a couple of `{client}-machine` users for old builds. We could probably rename one of those for this as they shouldn't be needed anymore.